src/components/NoteListing/NoteListing.jsx

```diff
@@ -54,20 +57,22 @@ export const NoteListing = ({ list, isPinSection, isTrash, isArchive }) => {
 	};
 
 	return list.map((note, i) => {
-		const { _id, title, body, noteColor, dateCreated, labels } = note;
+		const { _id, title, body, noteColor, dateCreated, labels, priority } = note;
 
 		const toggleArchiveBtn = isArchive ? bxIcons.archiveOut : bxIcons.archiveIn;
 		const toggleTrashBtn = isTrash ? <MdiRestore /> : bxIcons.trashAlt;
 		const togglePinBtn = isPinSection ? bxIcons.pinned : bxIcons.pin;
 
```

Comment:
Suggestion : Convert this into a component

![image](https://user-images.githubusercontent.com/79917843/161564404-87b1e447-a708-47ed-8050-c1f6854bb59f.png)